databricks/koalas/tests/test_dataframe.py

```diff
@@ -767,21 +768,23 @@ def check(op, right_kdf=right_kdf, right_pdf=right_pdf):
                                              suffixes=['_left', '_right']))
 
         # Test Series on the right
-        check(lambda left, right: left.merge(right), right_ks, right_ps)
-        check(lambda left, right: left.merge(right, left_on='x', right_on='x'),
-              right_ks, right_ps)
-        check(lambda left, right: left.set_index('x').merge(right, left_index=True, right_on='x'),
-              right_ks, right_ps)
-
-        # Test join types with Series
-        for how in ['inner', 'left', 'right', 'outer']:
-            check(lambda left, right: left.merge(right, how=how), right_ks, right_ps)
-            check(lambda left, right: left.merge(right, left_on='x', right_on='x', how=how),
+        # pd.DataFrame.merge with Series is implemented since version 0.24.0
+        if LooseVersion(pd.__version__) > LooseVersion("0.24.2"):
```

Comment:
nit: shall we use `0.24.0` if pandas supports since then?

Reply:
Sure.